### PR TITLE
0Box: Utility endpoints system tests 

### PR DESCRIPTION
**Pull request by iamrz1**
Resolves https://github.com/0chain/system_test/issues/547

### CODE REVIEW

File: tests/api_tests/0box_test.go [hunk 1]
```diff
@@ -597,6 +599,97 @@ func Test0Box(testSetup *testing.T) {
...
+	})
+
+	t.RunSequentially("Deleting FCM Token should work", func(t *test.SystemTest) {
+		csrfToken := createCsrfToken(t, zboxClient.DefaultPhoneNumber)
+		data, response, err := zboxClient.DeleteFCMToken(t, firebaseToken.IdToken, csrfToken, zboxClient.DefaultPhoneNumber)
+		require.NoError(t, err)
+		require.Equal(t, 200, response.StatusCode(), "Response status code does not match expected. Output: [%v]", response.String())
+		require.NotNil(t, data, "response object should not be nil")
+		require.Equal(t, "bolt", data.DeviceType, "response object should match input")
+	})
```
Comment (Kishan-Dhakan) [inline tests/api_tests/0box_test.go:692]: Can you add negative test cases as well?

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]
Happy to merge after negative cases are added

**Review by Kishan-Dhakan — APPROVED** [review]
LGTM

**Review by stewartie4 — APPROVED** [review]
LGTM

**Pull request merged**